python/llm/src/ipex_llm/transformers/convert.py

```diff
@@ -188,6 +188,12 @@ def is_linear_module(module):
     return result, (in_features, out_features, mp_group)
 
 
+def empty_cache_post(module, input, output):
+    try:
```

Comment:
why not check if it's xpu here?

Reply:
will it hurt performance for the xpu check? I will test it.

Reply:
Seems the impact is not obvious, https://github.com/analytics-zoo/nano/issues/1348#issuecomment-2102086271
using xpu check may be slightly slower than using this way, >5ms for 4k first token latency.

Do I need to change to xpu check?